ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -782,6 +784,12 @@ private static ConfigDef buildConfigDef(final ConfigGeneration generation) {
             KSQL_ACTIVE_PERSISTENT_QUERY_LIMIT_DEFAULT,
             Importance.MEDIUM,
             KSQL_ACTIVE_PERSISTENT_QUERY_LIMIT_DOC
+        ).define(
+            KSQL_FOREIGN_KEY_JOINS_ENABLED,
+            Type.BOOLEAN,
+            false,
```

Comment:
Not sure if this should be `true` for now? And we only disable by default is we don't make it?

Reply:
Feature flags guarding new functionality should default to false (especially for incompatible changes -- not as important for this particular feature but still nice to follow the same pattern) else we'd have to update/regenerate QTT tests while the feature is still a work-in-progress and likely to change.